pkgs/applications/editors/edit/default.nix

```diff
@@ -0,0 +1,38 @@
+{ stdenv, fetchgit, unzip, pkgconfig, ncurses, x11, cwebbin }:
```

Comment:
It works because `x11` is defined as an alias for `xlibsWrapper`; `libX11` won't work here, actually.